### ISSUE TITLE
Cloud Controller Mangles drains from the syslog_drain_urls endpoint that have commas

### ISSUE THREAD

**Issue by MasslessParticle** [cloudfoundry/cloud_controller_ng#1270]
Thanks for submitting an issue to `cloud_controller_ng`. We are always trying to improve! To help us, please fill out the following template.

## Issue

If a bound syslog drain has commas in the drain url (supported by some log consumers) the url is split and invalid drain urls are sent to callers.

## Context

A customer noticed an issue where the cf-syslog-drain-release was reporting parse errors and we tracked it to this endpoint.

## Steps to Reproduce

Bind a syslog drain with a url like:  https://hostname.com/tags/some,loggly,tags
Make a call to the `internal/v4/syslog_drain_urls` endpoint

## Expected result

```
    "drains": [
        "https://hostname.com/tags/some,loggly,tags"
      ],
```

## Current result

```
    "drains": [
        "https://hostname.com/tags/some",
        "loggly",
        "tags"
      ],
```

## Possible Fix

This line:
https://github.com/cloudfoundry/cloud_controller_ng/blob/65a75e6c97f49756df96e437e253f033415b2db1/app/controllers/internal/syslog_drain_urls_controller.rb#L39

splits the drains on commas.

## `name of issue` screenshot

[if relevant, include a screenshot]

**Comment by danhigham** [comment]
It turns out that you can encode just the commas and that suffice as a workaround;

`https://logs-01.loggly.com/inputs/<token>/tag/tag1%2Ctag2%2Ctag3/`

I would still expect it to be able to handle commas, perhaps the delimiter for drain URLs could be changed to a character that can't appear in a URL?

**Comment by cwlbraa** [comment]
fixed in aae32ff01
